### ISSUE TITLE
Prince of Persia II - MT-32 option incorrectly uses PC Speaker sound effects

### ISSUE THREAD

**Issue by M-Walrus** [flynnsbit/Top300_updates#78]
Should be using soundblasterPRO option along with mt-32 and strangely is using PC speaker currently instead.

* current workaround is to add a custom entry for "e:\games\princeoa\ppersia2\prince.exe" to the "-manually added games" menu and avoid the launcher overwriting your settings, after choosing the correct sound options in the setup for the game.

**Comment by diablonet** [comment]
We are aware of the situation, it is related to a sound blaster bug on the AO486 which freezes the game on the beach level and above, I can add another profile on the menu for MT32 music + Sound Effects (not SB but MT32).

**Comment by diablonet** [comment]
I got able to fix all the issues on this game, I posted the solution here: https://www.vogons.org/viewtopic.php?f=7&t=84387 

It will be fixed it by the TOP3000 update, but you may manually fix it on the meanwhile

**Comment by M-Walrus** [comment]
has this been updated in the pack yet? I ran the updater and still dont have a configuration option for prince of persia 2 for mt32+SB.

**Comment by diablonet** [comment]
I dont know if its on the update, but I made the fix, where you able to apply it?

**Comment by M-Walrus** [comment]
> I dont know if its on the update, but I made the fix, where you able to apply it?

No, I was waiting for the update that was mentioned and its been almost a year since you posted that so figured I was updating incorrectly or something

**Comment by diablonet** [comment]
I can help you with this, please DM me jorge_velazquez@hotmail.com, once I get a word from you, I will delete my email account
